src/styles/utils/emojiDefaultStyles/index.ts

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line no-restricted-imports
+import display from '@styles/utils/display';
+import type EmojiDefaultStyles from './types';
+
+const emojiDefaultStyles: EmojiDefaultStyles = {
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    ...display.dInlineFlex,
```

Comment:
Can you elaborate why do we need different style for native platform?

Reply:
@eh2077 `inline-flex` not supported in native, it will cause [your mentioned warning message](https://github.com/Expensify/App/pull/40617#discussion_r1576350360). the pattern inspired from [editedLabelStyles](https://github.com/Expensify/App/tree/main/src/styles/utils/editedLabelStyles).